packages/Meshfree/src/DTK_SplineOperator_def.hpp

```diff
@@ -301,6 +301,6 @@ void SplineOperator<DeviceType, CompactlySupportedRadialBasisFunction,
 #define DTK_SPLINE_OPERATOR_INSTANT( NODE )                                    \
     template class SplineOperator<typename NODE::device_type>;                 \
     template class SplineOperator<typename NODE::device_type, Wendland<0>,     \
-                                  MultivariatePolynomialBasis<Quadratic, 3>>;
+                                  MultivariatePolynomialBasis<Linear, 3>>;
```

Comment:
The two instantiations do the same. Just remove one of them.

Reply:
It's fine for me.